### ISSUE TITLE
Content stream "pop from empty list" - was: fails when running with docker

### ISSUE THREAD

**Issue by maxandersen** [jbarlow83/OCRmyPDF#121]
I have this error when running via docker:

```
docker run -t -i -v `pwd`:/home/docker jbarlow83/ocrmypdf "my\ test\ file.pdf" test.pdf
  ERROR - Traceback (most recent call last):
  File "/appenv/lib/python3.5/site-packages/ruffus/task.py", line 751, in run_pooled_job_without_exceptions
    register_cleanup, touch_files_only)
  File "/appenv/lib/python3.5/site-packages/ruffus/task.py", line 567, in job_wrapper_io_files
    ret_val = user_defined_work_func(*params)
  File "/appenv/lib/python3.5/site-packages/ocrmypdf/__main__.py", line 585, in repair_pdf
    pdfinfo.extend(pdf_get_all_pageinfo(output_file))
  File "/appenv/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 410, in pdf_get_all_pageinfo
    return [_pdf_get_pageinfo(infile, n) for n in range(pdf.numPages)]
  File "/appenv/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 410, in <listcomp>
    return [_pdf_get_pageinfo(infile, n) for n in range(pdf.numPages)]
  File "/appenv/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 392, in _pdf_get_pageinfo
    contentsinfo = _interpret_contents(contentstream)
  File "/appenv/lib/python3.5/site-packages/ocrmypdf/pageinfo.py", line 137, in _interpret_contents
    ctm = stack.pop()
IndexError: pop from empty list
```

any idea on how to fix that ?

**Comment by jbarlow83** [comment]
Does `docker run -t -i -v $(pwd):/home/docker jbarlow83/ocrmypdf --version` work and what is the output?

Can you send the PDF that caused this issue?

**Comment by jbarlow83** [comment]
Note: this is probably not a docker issue. The stack trace suggests that the PDF's content stream is invalid.

**Comment by maxandersen** [comment]
> Does docker run -t -i -v $(pwd):/home/docker jbarlow83/ocrmypdf --version work and what is the output?

`4.3.4`



> Can you send the PDF that caused this issue?

I could have sworn I tried this on a bunch of PDF files, but it doesn't fail now on some new pdf files.

I unfortunately can't share the one that is failing due to the nature of the document but I'll do if I find a less sensitive doc that fails.

Thanks.

**Comment by maxandersen** [comment]
I figured out what triggers the failure. 

It is on pdf's where I have used pdfpen pro to add annotations (I use that to mark which bills I payed etc.)

Here is a simple pdf file that just contains some text + overlay that ocrmypdf fails on.

[overlay.pdf](https://github.com/jbarlow83/OCRmyPDF/files/721923/overlay.pdf)

**Comment by jbarlow83** [comment]
Fixed in develop branch and will be released in v4.4. 

I've taken the liberty of adding your sample file to the test suite for future regression checks with credit to you for contributing it. If you don't wish to permit this please let me know and we'll find an alternative.

**Comment by maxandersen** [comment]
Great. I'm fine contributing that file to the project.
